docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+## How to contribute
+
+Pocket is an open source project, and as such we welcome any contribution from anyone on the internet.
+
+Please fork, code and submit a Pull Request for the team to review and merge. We ask that you please follow the
+guidelines below in order to submit your contributions for review:
+
+### High impact or architectural changes
+
+Reach out to us on [Telegram](https://t.me/POKTnetwork) and start a discussion with the team regarding your change
+before you start working. Communication is key for open source projects and asynchronous contributions.
+
+For an active research forum, checkout and post on [our forum](https://research.pokt.network).
+
+### Coding style
+
+- Code must adhere to the official Go formatting guidelines (i.e. uses [gofmt](https://golang.org/cmd/gofmt)).
+- (Optional) Use [Editor Config](https://editorconfig.org) to help your Text Editor keep the same formatting used
+  throughout the project.
+- Code must be documented adhering to the official Go commentary guidelines.
+- Pull requests need to be based on and opened against the `main` branch.
+
+### Criteria for PR Evaluation
+
+- Quality: the PR should include instructions on how to test the submitted code. Automated tests are not a must in all
```

Comment:
Note for a future PR.

We can create a template for the PR with headings (e.g. SUmmary, Purpose, Test, etc...) and add details for the user to fill out via comments.

Anything inside of `<!-- -->` is not evaluated by markdown,

Reply:
We could; the question of where to put the information, though, is partially answered by the very shape of this PR; testing information lives in the docs, for instance; summary and purpose should be tied to the origin document and the rest of the context that we'd provide in the goal. I think it's important that test information for each patch lives in the repo so that it can serve as a reference that lives with the repo, not necessarily get archived and forgotten with a closed PR.